From 8a0b4643e1119e994370089fd52721373e88bb51 Mon Sep 17 00:00:00 2001
From: William Brown <firstyear@redhat.com>
Date: Wed, 29 Mar 2017 10:59:14 +1000
Subject: [PATCH] Ticket 49196 - Autotune generates crit messages

Bug Description:  The cache sanity check generates critical messages.

Fix Description:  Make the sanity check generate warning messages.

https://pagure.io/389-ds-base/issue/49196

Author: wibrown

Review by: mreynolds (Thanks!)
---
 ldap/servers/slapd/back-ldbm/start.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/ldap/servers/slapd/back-ldbm/start.c b/ldap/servers/slapd/back-ldbm/start.c
index 759af8a..1ae9858 100644
--- a/ldap/servers/slapd/back-ldbm/start.c
+++ b/ldap/servers/slapd/back-ldbm/start.c
@@ -265,12 +265,12 @@ ldbm_back_start_autotune(struct ldbminfo *li) {
     issane = util_is_cachesize_sane(&total_cache_size);
     if (!issane) {
         /* Right, it's time to panic */
-        slapi_log_err(SLAPI_LOG_CRIT, "ldbm_back_start", "It is highly likely your memory configuration of all backends will EXCEED your systems memory.\n");
-        slapi_log_err(SLAPI_LOG_CRIT, "ldbm_back_start", "In a future release this WILL prevent server start up. You MUST alter your configuration.\n");
-        slapi_log_err(SLAPI_LOG_CRIT, "ldbm_back_start", "Total entry cache size: %lu B; dbcache size: %lu B; available memory size: %lu B; \n",
+        slapi_log_err(SLAPI_LOG_WARNING, "ldbm_back_start", "It is highly likely your memory configuration of all backends will EXCEED your systems memory.\n");
+        slapi_log_err(SLAPI_LOG_WARNING, "ldbm_back_start", "In a future release this WILL prevent server start up. You MUST alter your configuration.\n");
+        slapi_log_err(SLAPI_LOG_WARNING, "ldbm_back_start", "Total entry cache size: %lu B; dbcache size: %lu B; available memory size: %lu B; \n",
                   (PRUint64)total_cache_size, (PRUint64)li->li_dbcachesize, availpages * pagesize
         );
-        slapi_log_err(SLAPI_LOG_CRIT, "ldbm_back_start", "%s\n", msg);
+        slapi_log_err(SLAPI_LOG_WARNING, "ldbm_back_start", "%s\n", msg);
         /* WB 2016 - This should be UNCOMMENTED in a future release */
         /* return SLAPI_FAIL_GENERAL; */
     }
-- 
2.9.3

